tnefparse/cmdline.py

```diff
@@ -47,18 +49,16 @@
          help="extract a json dump of the tnef contents")
 
 
-def tnefparse() -> None:
+def tnefparse(argv: typing.List = None) -> None:
     "command-line script"
 
-    if len(sys.argv) == 1:
+    try:
+        args = parser.parse_args(argv)
+    except argparse.ArgumentError:
```

Comment:
Unfortunately, this does work as before, as argparse does not throw an `ArgumentError` and instead prints a very brief help text instead of the desired one, also see my comment here: 
https://github.com/koodaamo/tnefparse/pull/84#discussion_r532005950

Reply:
@eumiro Do you plan to continue the work on this PR or you do you want me to finish it?